src/language-js/print/estree.js

```diff
@@ -101,7 +102,11 @@ function printEstree(path, options, print, args) {
     case "JsonRoot":
       return [print("node"), hardline];
     case "File":
-      return printHtmlBinding(path, options, print) ?? print("program");
+      return (
+        printHtmlBinding(path, options, print) ??
+        printVueScriptGenericParameters(path, options, print) ??
```

Comment:
Let's put it in `printHtmlBinding` like function parameters do.

Reply:
I hope I understood it properly, I moved whole `printVueScriptGenericParameters` inside `printHtmlBinding`. Is this right?

Reply:
That's correct, thank you.